### PR TITLE
#18639 exclude archived sites

### PR DESCRIPTION
**Pull request by fabrizzio-dotCMS**
This comes to solve https://github.com/dotCMS/core/issues/18639
Basically we're now excluding any site marked as archived (deleted on the index).

### CODE REVIEW

File: dotCMS/src/main/java/com/dotcms/rest/api/v1/apps/SiteViewPaginator.java [hunk 1]
```diff
@@ -22,6 +24,7 @@
 import java.util.Map;
 import java.util.Objects;
 import java.util.Set;
+import java.util.function.Predicate;
```
Comment (dev-dotcms) [inline dotCMS/src/main/java/com/dotcms/rest/api/v1/apps/SiteViewPaginator.java:27]: Codacy found an issue: [Avoid unused imports such as 'java.util.function.Predicate'](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5788531)

File: dotCMS/src/main/java/com/dotcms/security/apps/AppsAPIImpl.java [hunk 2]
```diff
@@ -50,6 +53,7 @@
 import java.util.Map.Entry;
 import java.util.Optional;
 import java.util.Set;
+import java.util.function.Predicate;
```
Comment (dev-dotcms) [inline dotCMS/src/main/java/com/dotcms/security/apps/AppsAPIImpl.java:56]: Codacy found an issue: [Avoid unused imports such as 'java.util.function.Predicate'](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5788531)

File: dotCMS/src/main/java/com/dotcms/rest/api/v1/apps/SiteViewPaginator.java [hunk 3]
```diff
@@ -5,12 +5,14 @@
 import com.dotcms.util.pagination.PaginationException;
 import com.dotcms.util.pagination.PaginatorOrdered;
 import com.dotmarketing.beans.Host;
+import com.dotmarketing.business.APILocator;
```
Comment (dev-dotcms) [inline dotCMS/src/main/java/com/dotcms/rest/api/v1/apps/SiteViewPaginator.java:8]: Codacy found an issue: [Avoid unused imports such as 'com.dotmarketing.business.APILocator'](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5788531)

File: dotCMS/src/main/java/com/dotcms/rest/api/v1/apps/SiteViewPaginator.java [hunk 4]
```diff
@@ -5,12 +5,14 @@
...
 import com.dotmarketing.common.model.ContentletSearch;
 import com.dotmarketing.exception.DotDataException;
 import com.dotmarketing.exception.DotRuntimeException;
 import com.dotmarketing.exception.DotSecurityException;
 import com.dotmarketing.portlets.contentlet.business.ContentletAPI;
 import com.dotmarketing.portlets.contentlet.business.HostAPI;
+import com.dotmarketing.portlets.contentlet.model.Contentlet;
```
Comment (dev-dotcms) [inline dotCMS/src/main/java/com/dotcms/rest/api/v1/apps/SiteViewPaginator.java:15]: Codacy found an issue: [Avoid unused imports such as 'com.dotmarketing.portlets.contentlet.model.Contentlet'](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5788531)

**Review by nollymar — APPROVED** [review]

**Review by jdotcms — APPROVED** [review]

**Review by dsilvam — APPROVED** [review]

**Review by erickgonzalez — APPROVED** [review]

**Pull request merged**